doc/src/manual/conversion-and-promotion.md

```diff
@@ -111,20 +111,12 @@ example, since the type is a singleton, there would never be any reason to use i
 the body. When invoked, the method determines whether a numeric value is true or false as a boolean,
 by comparing it to one and zero:
 
-```julia
+```jldoctest bool
 julia> convert(Bool, 1)
 true
 
 julia> convert(Bool, 0)
 false
-
-julia> convert(Bool, 1im)
-ERROR: InexactError()
- in convert(::Type{Bool}, ::Complex{Int64}) at ./complex.jl:23
- ...
-
-julia> convert(Bool, 0im)
-false
 ```
 
 The method signatures for conversion methods are often quite a bit more involved than this example,
```

Comment:
This section makes no sense. The implementation above *is* the exact implementation: https://github.com/JuliaLang/julia/blob/master/base/bool.jl#L7

And the next section:

> This is the actual implementation in Julia:

and then an example with conversion to real from complex is stated instead of real to bool as above.